scripts/apps/archive/services/FamilyService.ts

```diff
@@ -102,6 +103,16 @@ export function FamilyService(api, desks) {
         return query(filter, 'versioncreated', 'asc');
     };
 
+    /**
+     * Fetch related items
```

Comment:
The comment is made of 3 words, 2 of which are the in the function name. If that extra word is relevant, add it to the function name instead of having it as a comment duplicating the function name.